include/linalg.hpp

```diff
@@ -2103,18 +2108,31 @@ namespace cytnx {
     //===========================================
     /**
     @brief perform matrix exponential for Hermitian matrix
-    @param in input Tensor, should be Hermitian
-    @param a rescale factor
-    @param b bias
+    @details This function perform matrix exponential for Hermitian matrix, That is,
+    \f[
+        O = \exp{(aM + b)}
+    \f]
+    @param[in] in input Tensor, should be Hermitian
+    @param[in] a rescale factor
+    @param[in] b bias
     @return
         [Tensor]
-
-    description:
-        perform matrix exponential with \f$O = \exp{aM + b}\f$.
+    @pre the \p in should be a Hermitian matrix.
+    @warning If \p in is not a Hermitian matrix, the result will be undefined.
```

Comment:
I suggest:
```
If `p` is not a Hermitian matrix, only the lower triangular matrix will be used. (This is strongly not recommended. use ExpM[link to ExpM] instead)
```

Also since ExpM and ExpH both use Eig and Eigh you might need to do modifications there as well

Reply:
remember to modify the Eigh as well (only lower triangular will be used)